e2e/test/svelte-jest-project/package.json

```diff
@@ -0,0 +1,24 @@
+{
+  "name": "svelte-jest-project",
+  "private": true,
+  "version": "0.0.0",
+  "scripts": {
+    "test": "stryker run",
+    "posttest": "mocha --no-config --no-package --timeout 0 verify/verify.js"
+  },
+  "devDependencies": {
+    "@babel/preset-env": "^7.19.4",
+    "@sveltejs/vite-plugin-svelte": "^1.1.0",
+    "@testing-library/jest-dom": "^5.16.5",
+    "@testing-library/svelte": "^3.2.2",
+    "@testing-library/user-event": "^14.4.3",
+    "babel-jest": "^29.2.2",
+    "babel-plugin-transform-vite-meta-env": "^1.0.3",
+    "jest": "^29.2.2",
+    "jest-environment-jsdom": "^29.2.2",
+    "jest-transform-stub": "^2.0.0",
+    "svelte": "^3.52.0",
+    "svelte-jester": "^2.3.2",
+    "vite": "^3.2.0"
```

Comment:
Can we add these dependencies to the `e2e/package.json` file instead? That makes for a faster CI pipeline.